src/pypi_simple/client.py

```diff
@@ -113,12 +114,18 @@ def get_index_page(
 
             ``accept`` parameter added
 
+        .. versionchanged:: 1.4.2
+
+            ``headers`` parameter added
+
         :param timeout: optional timeout to pass to the ``requests`` call
         :type timeout: float | tuple[float,float] | None
         :param Optional[str] accept:
             The :mailheader:`Accept` header to send in order to
             specify what serialization format the server should return;
             defaults to the value supplied on client instantiation
+        :param Optional[Dict[str, str]] headers:
```

Comment:
```suggestion
        :param Optional[Dict[str,str]] headers:
```

I believe Sphinx requires the type in a `:param:` to be a single word.

Reply:
I ran `tox -e docs` and it seems to be fine with this (after `s/Dict/dict/`)?

![image](https://github.com/jwodder/pypi-simple/assets/49834/ddadd5e5-d3e1-46f9-877a-274f12629540)

Reply:
Huh.  Never mind, then.